src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -107,8 +106,10 @@ public function handle(GetResponseEvent $event)
                 $this->securityContext->setToken($authenticatedToken);
 
                 if (null !== $this->dispatcher) {
-                    $event = new InteractiveLoginEvent($request, $authenticatedToken);
-                    $this->dispatcher->dispatch(SecurityEvents::INTERACTIVE_LOGIN, $event);
+                    $this->dispatcher->dispatch(
+                        new InteractiveLoginEvent($request, $authenticatedToken),
```

Comment:
Do you know if there's a particular reason that [InteractiveLoginEvent](https://github.com/symfony/symfony/blob/v4.3.4/src/Symfony/Component/Security/Http/Event/InteractiveLoginEvent.php) doesn't extend [`Symfony\Contracts\EventDispatcher\Event`](https://github.com/symfony/symfony/blob/v4.3.4/src/Symfony/Contracts/EventDispatcher/Event.php) given that `Symfony\Component\EventDispatcher\Event` is deprecated? I suppose it's outside of the scope of this PR (and library), but perhaps @nicolas-grekas can answer this?

Reply:
for BC I suppose: changing the inheritance chain would break it.

Reply:
@nicolas-grekas: Ah, that makes complete sense and is exactly what I observed about this library's AlreadyAuthenticatedEvent in https://github.com/jmikola/AutoLogin/pull/21#discussion_r327872762. Pardon my brain fart, but thanks for the quick reply!